tests/publiccloud/img_proof.pm

```diff
@@ -112,14 +112,15 @@ sub run {
     } else {
         $provider = $self->provider_factory();
         $instance = $provider->create_instance();
+
+        $instance->wait_for_guestregister() if is_ondemand();
```

Comment:
now you moved it back into "not QAM" branch , any particular reason for this ?

Reply:
Yes. In latest we create and register the instance directly in `img_proof` test module. But in QAM we have separate modules for that.